cmd/tailscale/backend.go

```diff
@@ -121,16 +121,27 @@ func newBackend(dataDir string, jvm *jni.JVM, appCtx jni.Object, store *stateSto
 		return nil, fmt.Errorf("runBackend: NewUserspaceEngine: %v", err)
 	}
 	b.logIDPublic = logID.Public().String()
-	if err := startNetstack(log.Printf, dialer, engine); err != nil {
-		return nil, fmt.Errorf("startNetstack: %w", err)
+	tunDev, magicConn, ok := engine.(wgengine.InternalsGetter).GetInternals()
+	if !ok {
+		return nil, fmt.Errorf("%T is not a wgengine.InternalsGetter", engine)
+	}
+	ns, err := netstack.Create(logf, tunDev, engine, magicConn, dialer)
+	if err != nil {
+		return nil, fmt.Errorf("netstack.Create: %w", err)
 	}
-	local, err := ipnlocal.NewLocalBackend(logf, b.logIDPublic, store, dialer, engine, 0)
+	ns.ProcessLocalIPs = false
```

Comment:
While you're here, can you add comments on why we enable subnets but not local IPs?

Reply:
done